scripts/unzip_backup.py

```diff
@@ -55,13 +55,20 @@ def get_history_file():
 def push_csv_to_db(extracted_csv_path):
     rowcount = 0
     manifest = json.dumps(dict())
+    _, _, files = next(os.walk(extracted_csv_path))
+    if len(files) not in (5, 9):
+        print('[ERROR] the number of log tables in uploaded report does not match preset values')
+        print('{} has length {}'.format(files, len(files)))
+        exit(1)
+    turn_of_the_month = len(files) > 5
```

Comment:
see the comment above: it seems like 5 is a magic number - maybe a constant variable that's initialize to 5 can be used here instead to improve readabiliy?